web-app/src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -56,13 +57,22 @@ const BookingModalForm = props => {
       let error = {
         message: `You should give ${daysNotice} working/business days notice to request ${bookingDuration} ${
           bookingDuration > 1 ? 'days' : 'day'
-        } off, and therefore your request might be declined.`,
+          } off, and therefore your request might be declined.`,
       };
       return fromTodayToStartDateRequested < daysNotice ? (
         <Errorbox error={error} label="Warning" />
       ) : null;
     }
   };
+  //paul come back here
```

Comment:
Don't do this. If there's something that will need to be looked at in the future raise a ticket for it. You may not be on the project and there could be something required here that no-one else knows about.